.travis.yml

```diff
@@ -59,6 +59,7 @@ env:
 install:
   - docker rmi $(docker images -q) || true
   - travis_retry make install-ci
+  - bash ./scripts/travis/upgrade-docker.sh
```

Comment:
Hi @morlay 

We started moving to Github Actions due to travis credits expiration. As you're already aware of official docker's buildx action: https://github.com/docker/setup-buildx-action I am pretty sure it will make the PR lot simpler. Sorry for the trouble caused by this.

Reply:
mv to github actions will be simple 🎉
as i setup for jaeger operator

Reply:
@Ashmita152 I removed codes to setup buildx in travis.

consider setup two release workflows. 

one for base images (could set  `on.push.paths: docker/**/Dockerfile`)

```
make dockerx-baseimg-debugimg
```

another for components.
```
make dockerx-push
```

Reply:
Sure I will do it today.